notebooks/TFSegFormer.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<p>Why is there there bullet point beside TensorFlow? 🧐</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/segformer-tf-transformers/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>